source/index.js

```diff
@@ -435,7 +435,6 @@ host.BrowserHost = class {
     }
 
     _openModel(url, identifier) {
-        url = url + ((/\?/).test(url) ? "&" : "?") + (new Date()).getTime();
```

Comment:
@JarnoRFB See [Bypassing the cache](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/Using_XMLHttpRequest#bypassing_the_cache). Fix either the query parameter parser or by using a supported pattern.